eos/form-factors/analytic-b-to-pi.cc

```diff
@@ -173,8 +173,8 @@ namespace eos
             const double mb = this->m_b_msbar(mu), mb2 = mb * mb, mb4 = mb2 * mb2;
             const double Mprime4 = Mprime2 * Mprime2;
 
-            const double cond_qq_mu = -fpi * fpi * this->pi.mupi(mu) / 2.0; // <qq>@mu
-            const double cond_qq_1 = -fpi * fpi * this->pi.mupi(1.0) / 2.0; // <qq>@1GeV
+            const double cond_qq_mu = -fpi * fpi * this->lcdas->mu3(mu) / 2.0; // <qq>@mu
```

Comment:
Can you put parenthesis after `mu` to avoid the cast? or remove them 3 lines below, I don't know...

Reply:
Hm. This is a purely aesthetic issue, since the cast and the evaluations are achieving the same thing: evaluating the current value of the parameter. However, I agree that it should be consistent.